index.js

```diff
@@ -504,6 +510,12 @@ class Flag {
     return this
   }
 
+  choices (values) {
+    this.valueChoices = values.split(',').map(s => s.trim())
+    this.description += ` (choices: ${values})`
```

Comment:
```suggestion
    this.valueChoices = values
    this.description += ` (choices: ${values.join(', ')})`
```

No need for string parsing, `values` can just be an array of strings.

Reply:
updated